api/features_api.py

```diff
@@ -348,11 +348,16 @@ def do_patch(self, **kwargs):
     if not feature:
       self.abort(400, msg=f'Feature not found for ID {feature_id}')
 
-    # Validate the user has edit permissions and redirect if needed.
-    redirect_resp = permissions.validate_feature_edit_permission(
-        self, feature_id)
-    if redirect_resp:
-      return redirect_resp
+
+    # If submitting an OT creation request, the user must have feature edit
```

Comment:
Where's the if-statement condition that narrows this to just OT creation requests?

Reply:
I'm going to update this to use the Stages API. It's difficult to discern when an OT creation request comes in, and the logic starts to get muddied here in the Features API. I'll update you when it's ready 🙂

Reply:
Okay, I've updated the logic to work in the Stages API rather than the Features API. 👍